test/Microsoft.Extensions.Internal.Test/ObjectMethodExecutorTest.cs

```diff
@@ -427,6 +511,17 @@ public ValueTask<string> ValueTaskOfReferenceType(string result)
             public void MethodWithMultipleParameters(int valueTypeParam, string referenceTypeParam)
             {
             }
+
+            public FSharpAsync<string> FSharpAsyncMethod(string parameter)
```

Comment:
Thanks, @SteveSandersonMS! I added a comment to show an `async` member, though I doubt you needed that assistance. The F# looks good. Thanks again for pursuing this, and please let me know if I can help by adding test cases. I'm trying to get the Mvc project building locally so that I can do so.